_posts/en/newsletters/2020-02-19-newsletter.md

```diff
@@ -0,0 +1,298 @@
+
+- **Decoy nodes and lightweight rendez-vous routing:** Bastien
+  Teinturier previously [posted][teinturier delink] about breaking the
+  link between what data is included in a [BOLT11][] invoice and the
+  funding transaction of the channel that will receive the payment (see
+  [Newsletter #82][news82 unannounced]).  After further discussion and
+  refinement, Teinturier [noted][teinturier rv] a side effect of his
+  scheme might enable convenient rendez-vous routing---privacy-enhanced
+  payment routing where neither the receiving node nor the spending node
+  learns anything about each other's network identity.  For more information, see
+  [Teinturier's documentation][rv gist] for the scheme, read about
+  previous discussion of rendez-vous routing in [Newsletter #22][news22
+  rv], and review discussion of the topic in Monday's LN developer
+  [specification meeting][spec meet].
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+- **BTCPay Vault using HWI for signing:** [BTCPay Vault][btcpay vault blog] is
+a desktop application that uses [HWI][topic hwi] to coordinate signing
+transactions with a variety of hardware wallets. While BTCPay Server created
+BTCPay Vault, the software can be repuposed for use in other applications.
+
+- **CKBunker using PSBTs for an HSM:** [CKBunker][coinkite bunker] allows users
+to configure rule-based spending conditions for an online, Tor-enabled Coldcard
+hardware wallet. The Coldcard then functions like an HSM, signing
+[PSBTs][topic psbt] delivered via a Tor hidden service.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #18104][] ends support for building 32-bit x86 binaries
+  for Linux as part of the Bitcoin Core release process.  The
+  corresponding 32-bit binaries for Windows were previously removed
+  several months ago (see [Newsletter #46][news46 win32]).  The 32-bit
+  Linux binaries are still built as part of Bitcoin Core's continuous
+  integration tests and users may still build them manually, but the
+  binaries are no longer being distributed by the project due to a
+  lack of use and hands-on developer testing.
+
+- [C-Lightning #3488][] standardizes C-Lightning’s requests for Bitcoin data
+  making it possible to run C-Lightning on something other than Bitcoin Core
+  as the backend. This pull request is part of a larger project to allow more
+  freedom for how C-Lightning interacts with the Bitcoin backend as proposed
+  in [C-Lightning #3354][]. Keeping the backend interactions general and
+  unassuming allows for plugins to either make standard RPC calls, combine
```

Comment:
I don't think unassuming (which usually means 'modest' or 'unpretentious') is quite the right word here. I think 'general and unassuming' could be replaced with 'generic'